proxy/http/HttpTransact.cc

```diff
@@ -3460,7 +3460,7 @@ HttpTransact::HandleStatPage(State *s)
       s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, s->internal_msg_buffer_type, len);
     }
   } else {
-    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 9);
+    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 10);
```

Comment:
Maybe clang-tidy can do it? Or it is possible to write a custom check?

I often use a ``lengthof`` template to do the static strlen:

```C
template <unsigned>
size_t lengthof(const char (&)[N]) {
    return N - 1;
}
```